### ISSUE TITLE
Can not change language

### ISSUE THREAD

**Issue by fs1995** [giuspen/cherrytree#2270]
**Version, Operative system**
Version 0.99.55
Linux Mint 21.1 (Flatpak and AppImage)
Pop!_OS 22.04 (Flatpak)
Fedora Kinoite 38 (Flatpak and AppImage)

**Describe the bug**
CherryTree is only displayed in English, regardless of language selected. Even when selected "System Default", and my system language is not English, CherryTree is in English.

**To Reproduce**
Flatpak and AppImage versions seem to be stuck in English.

**Comment by fs1995** [comment]
Here is the error I get when launching:

```
[silver@Silver-5580 Downloads]$ ./CherryTree-0.99.55-x86_64.appimage 
** Message: 16:56:28.874: Language overwrite = de (localedir = /usr/share/locale)

(process:3831): Gtk-WARNING **: 16:56:28.944: Locale not supported by C library.
        Using the fallback 'C' locale.
```

**Comment by giuspen** [comment]
I will try to solve before the next release, I'm busy at the moment completing the biggest feature that will land in 0.99.56 (support for a new multifile storage)

**Comment by giuspen** [comment]
I can see that the AppImage is not working but the flatpak seems to be working for me.
What I get is: `** Message: 23:51:16.552: Language overwrite = it (localedir = /app/share/locale)`
Can you show me also what you get from the flatpak too?

**Comment by fs1995** [comment]
I have live booted Ubuntu 23.04 to do the following testing:

I installed flatpak normally (as system), and CherryTree language does not change from English. I get the same error on launch:

```
** Message: 00:44:48.240: Language overwrite = de (localedir = /app/share/locale)
 
(process:2): Gtk-WARNING **: 00:44:48.243: Locale not supported by C library.
	Using the fallback 'C' locale.
```

I installed flatpak again as user (with the --user flag), and I can now change the language! I selected several randomly, and seems to all work.

AppImage still can not change language.
